.github/workflows/build-unsigned-archive.yml

```
@@ -32,22 +35,47 @@ jobs:
+
+          if [ -z "$APPLE_ID" ]; then
+            echo "Error: APPLE_ID is not set."
+            exit 1
+          else
+            echo "APPLE_ID is set."
+          fi
 
       - name: Build Release Version (Unsigned) via Makefile
-        run: make archive
+        run: make build-release
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)